budgets.json

```diff
@@ -0,0 +1,23 @@
+[{
+    "resourceSizes": [{
+            "resourceType": "script",
+            "budget": 1000
```

Comment:
These are way over what they should be and placeholders.

Reply:
Is this the transferred size of the files or the unpacked size?

Reply:
[Transfer size](https://developers.google.com/web/tools/lighthouse/audits/budgets)